packages/snaps-controllers/src/snaps/endowments/eip1193.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  EndowmentGetterParams,
+  PermissionSpecificationBuilder,
+  PermissionType,
+  ValidPermissionSpecification,
+} from '@metamask/controllers';
+import { SnapEndowments } from './enum';
+
+const permissionName = SnapEndowments.EIP1193;
```

Comment:
I'm not sure if I like the name of this permission. EIP-1193 makes sense to us, but the average developer may not know what it is. Something like `endowment:ethereum-provider` might be better.

Reply:
@rekmarks

Reply:
Yeah, `endowment:ethereum-provider` is probably better. The comments and documentation should clearly state that it's implementing EIP-1193, and does not contain legacy features shipped to dapps.